src/styles/itemAttributes.ts

```diff
@@ -125,21 +129,23 @@ export class ItemAttributes {
    */
   public static getAutomaticRangeStyleLinear(
     value: number,
-    {max, min}: {max: number; min: number},
+    {max, min}: IStyleAutoRange,
     lower: EdgeWidthExtrema | NodeSizeExtrema,
     higher: EdgeWidthExtrema | NodeSizeExtrema
-  ): string {
-    // apply default style when min equal max
-    if (max === min || isNaN(value)) {
-      return '100%';
-    }
+  ): string | undefined {
+    if (max !== undefined && min !== undefined) {
+      // apply default style when min equal max
+      if (max === min || isNaN(value)) {
```

Comment:
is it new code? why adding this?

Reply:
if yes, we'll some validation (QA), but we should avoid it in lint PRs

Reply:
Nope it's not